dev/com.ibm.ws.jca.utils/resources/com/ibm/ws/jca/utils/internal/resources/JcaUtilsMessages.nlsprops

```diff
@@ -168,28 +168,36 @@ J2CA9936.default.value.error.useraction=Review the error and adjust the resource
+J2CA9942.cfgprop.invalid.parameters.useraction=Investigate the vendor documentation and problem determination processes for the resource adapter.
 
 J2CA9943.missing.connector.dd=J2CA9943E: The resource adapter {0} is invalid. It should contain either a deployment descriptor or a JavaBean class annotated with one of the resource adapter annotations.
-J2CA9943.missing.connector.dd.explanation=A valid resource adapter must have either a deployment descriptor or a JavaBean class annotated with one of @Connector, @AdministeredObject, @ConnectionDefinition. @ConnectionDefinitions or @Activation annotations.
-J2CA9943.missing.connector.dd.useraction=Investigate the resource adapter vendor's documentation and problem determination processes.
+J2CA9943.missing.connector.dd.explanation=A valid resource adapter must have either a deployment descriptor or a JavaBean class that is annotated with one of the following annotations: @Connector, @AdministeredObject, @ConnectionDefinition, @ConnectionDefinitions, or @Activation.
+J2CA9943.missing.connector.dd.useraction=Investigate the vendor documentation and problem determination processes for the resource adapter.
+
+J2CA9944.missing.connector.jakarta=J2CA9944E: The {0} resource adapter can contain either a deployment descriptor or a JavaBeans class that is annotated with one of the resource adapter annotations from the Jakarta Connectors specification. If the resource adapter uses classes from the Java EE Connector Architecture 1.7 or lower version specification (javax.resource.spi package), use the jca-1.7 or lower version feature rather than Jakarta EE features.
+J2CA9944.missing.connector.jakarta.explanation=The resource adapter must have either a deployment descriptor or a JavaBean class that is annotated with one of the following annotations from the jakarta.resource.spi package: @Connector, @AdministeredObject, @ConnectionDefinition, @ConnectionDefinitions, or @Activation.
+J2CA9944.missing.connector.jakarta.useraction=Use the jca-1.7 or lower version feature with a resource adapter that implements the Java EE Connector Architecture specification.
+
+J2CA9945.missing.connector.javax=J2CA9945E: The {0} resource adapter must have either a deployment descriptor or a JavaBeans class that is annotated with one of the resource adapter annotations from the Java EE Connector Architecture specification. If the resource adapter uses classes from the Jakarta Connectors 2.0 or higher version specification (jakarta.resource.spi package), use the connectors-2.0 or higher version feature rather than Java EE features.
+J2CA9945.missing.connector.javax.explanation=The resource adapter must have either a deployment descriptor or a JavaBean class that is annotated with one of the following annotations from the javax.resource.spi package: @Connector, @AdministeredObject, @ConnectionDefinition, @ConnectionDefinitions, or @Activation.
```

Comment:
JavaBean --> JavaBeans

Reply:
@dmuelle sorry for missing those. The JavaBeans terminology is fixed in https://github.com/OpenLiberty/open-liberty/pull/15721/commits/d6b8ca2d08127caea7ff72abeb746944cd5cf5df